code/modules/emotes/emote_sounds.dm

```diff
@@ -0,0 +1,52 @@
+/*
+* Declare for storing species and gender specific emote sounds and make them overridable between species. 
+* Also the soundlist is loaded from a json array that sorts lists of sounds by emote name and optionally gender.
+*/
+
+//Mainly used for debugging and verifying everything loaded properly
+var/global/list/decl/emote_sounds/all_emote_decl = list()
+
+//merging proc for the merge assoc list proc
+/proc/merge_replace_if_not_null(var/a, var/b)
+	return b? b : a
+
+//A set of sounds for a set of emotes
+/decl/emote_sounds
+	var/list/soundbank = list() // 2d array, first is emote_name, second level contains sounds list by gender, or just a soundlist if there's no need for gender
+
+/decl/emote_sounds/New()
```

Comment:
Please avoid `New()`, `Initialize()` exists for decls and is preferred.

Reply:
Gotcha, will change that.